src/serializers/index.js

```diff
@@ -8,6 +8,9 @@ import opWrapper from './protocol/opWrapper';
 
 export { default as transaction, signedTransactionSerializer as signedTransaction } from './transaction';
 
+export { approvalDelta } from './wallet';
+export { accountListing } from './protocol/account/index';
```

Comment:
export `accountListing` in `protocol/index` (like `price` of asset)
rm from here

Reply:
`AccountListingSerializer` and `ACCOUNT_LISTING` too